src/pages/home.js

```diff
@@ -1,11 +1,21 @@
-import React from "react";
-
+import React, { useState } from "react";
 import { Container } from "react-bootstrap";
+import LeftNavigation from "../components/left-navigation";
 import Overview from "../components/overview";
 
 function Home() {
+  const [showNav, setShowNav] = useState(false);
+  const handleClose = () => setShowNav(false);
+  const handleShow = () => setShowNav(true);
+
   return (
     <Container>
+      <LeftNavigation show={showNav} s onHide={handleClose} />
+
+      <button className="btn btn-primary" onClick={handleShow}>
```

Comment:
you can use this icon in dark gray color and position it to the left (with some proper margins) and no background.

```
import { FaBarsStaggered } from "react-icons/fa6";
```